### PR TITLE
HH-150/HH-151: placeholder image

### PR DESCRIPTION
**Pull request by nenonja**
## Description
- Add placeholder (background fallback) image to Cards, Hero and Page
- Customizable placeholser image(s) can be added to RHHC config : (see another pr https://github.com/City-of-Helsinki/react-helsinki-headless-cms/pull/43)

## Issues
https://helsinkisolutionoffice.atlassian.net/browse/HH-150
https://helsinkisolutionoffice.atlassian.net/browse/HH-151
### Closes

**[DEV-XXX](https://helsinkisolutionoffice.atlassian.net/browse/DEV-XXX):**

### CODE REVIEW

File: src/domain/search/landingPage/LandingPage.tsx [hunk 1]
```diff
@@ -17,9 +17,8 @@ export function LandingPageContentLayout({
 }: LandingPageProps & PageContentLayoutProps) {
   const { title, description, heroLink } = landingPage?.translation || {};
   const heroImage =
-    landingPage?.desktopImage?.edges &&
-    landingPage?.desktopImage?.edges[0]?.node?.mediaItemUrl;
-
+    landingPage?.desktopImage?.edges?.[0]?.node?.mediaItemUrl ?? undefined;
+  console.log('heroImage', heroImage);
```
Comment (nikomakela) [inline src/domain/search/landingPage/LandingPage.tsx:21]: console.log!?

**Review by nikomakela — APPROVED** [review]
LGTM, but I haven't seen the changes of RCHC yet.

File: src/domain/search/landingPage/LandingPage.tsx (hunk 1 above)
Comment (nenonja) [reply src/domain/search/landingPage/LandingPage.tsx:21]: We should  have eslint to prevent these. I can add that rule too.

**Comment by nenonja** [comment]
> LGTM, but I haven't seen the changes of RCHC yet.
> 
> There is 1 console.log that should be removed

RHHC pr is here if you would like to check it: https://github.com/City-of-Helsinki/react-helsinki-headless-cms/pull/43

**Pull request merged**